client/hosting/overview/components/migration-overview/components/cards.tsx

```diff
@@ -0,0 +1,47 @@
+import { translate } from 'i18n-calypso';
+import { HostingCard, HostingCardGrid } from 'calypso/components/hosting-card';
+
+export const cards = [
```

Comment:
I think we need to name this better. I don't feel confident using casing to differentiate constants. If you look below, we are setting Cards. 

Beyond that, I also think we should add a more explicit name, as this is too generic.